backend/src/main/scala/cromwell/backend/standard/StandardLifecycleActorFactory.scala

```diff
@@ -181,12 +182,20 @@ trait StandardLifecycleActorFactory extends BackendLifecycleActorFactory {
 
   override def getExecutionRootPath(workflowDescriptor: BackendWorkflowDescriptor, backendConfig: Config,
                                     initializationData: Option[BackendInitializationData]): Path = {
-    initializationData.get.asInstanceOf[StandardInitializationData].workflowPaths.executionRoot
+    initializationData match {
+      // Try to use the initializationData if present to avoid recreating a WorkflowPath
+      case Some(data) => data.asInstanceOf[StandardInitializationData].workflowPaths.executionRoot
+      case None => new WorkflowPathsWithDocker(workflowDescriptor, backendConfig).executionRoot
+    }
   }
 
   override def getWorkflowExecutionRootPath(workflowDescriptor: BackendWorkflowDescriptor, backendConfig: Config,
                                             initializationData: Option[BackendInitializationData]): Path = {
-    initializationData.get.asInstanceOf[StandardInitializationData].workflowPaths.workflowRoot
+    initializationData match {
+        // Try to use the initializationData if present to avoid recreating a WorkflowPath
+      case Some(data) => data.asInstanceOf[StandardInitializationData].workflowPaths.workflowRoot
+      case None => new WorkflowPathsWithDocker(workflowDescriptor, backendConfig).workflowRoot
```

Comment:
Since this safety catch appears twice, is it worth making the backup `WorkflowPathsWithDocker` a `lazy val`?